packages/flutter_tools/test/general.shard/commands/build_appbundle_test.dart

```diff
@@ -155,6 +155,8 @@ void main() {
       });
 
       mockAndroidSdk = MockAndroidSdk();
+      when(mockAndroidSdk.licensesAvailable).thenReturn(true);
+      when(mockAndroidSdk.platformToolsAvailable).thenReturn(true);
```

Comment:
Oh no.  This whole test uses the local file system.  Going to see if we can get it to use memory fs instead.

Reply:
This test can't use memory FS because it eventually uses gradle to interact with the file system.

Gradle can do all sorts of weird things to your local android SDK.  

Most of these tests should probably be moved to the permeable shard or the integration shard.

I think it makes sense to land this change for now so we can start randomizing the tests and avoid more weird regressions popping up in the mean time.